README.md

```diff
@@ -465,6 +467,76 @@ df.drop('total', axis=1)
 </details>
 
 
+### Calendar
+
+#### [Holidays](./holiday) -- Collect Holiday, Workday Data
+
+<details>
+  <summary>What are the supported countries and their country code and languages supported by this api?</summary>
```

Comment:
What are the supported countries and their country code and languages supported by this api? ---> What are the supported countries, their country codes and languages supported?